client/css/reviewRating.scss

```diff
@@ -106,7 +110,7 @@
   border: 1px solid black;
   height: fit-content;
   max-width: 80%;
-  min-width: 30%;
+  min-width: 40%;
```

Comment:
I still think percentages are difficult to calculate. I'm a fan of using pixels or ems and then using media queries to account for the different screen sizes.

Reply:
You mentioned using media queries yesterday and I agree with that. But I will keep that one for now since it still looks good on the page. thanks